shell.py

```diff
@@ -144,11 +153,11 @@ def init_printing(*args, **kwargs):
 
 
 # The blueprint used to store user queries
-class Searches(db.Model):
-    user_id = db.UserProperty()
-    query = db.StringProperty(multiline=True)
-    timestamp = db.DateTimeProperty(auto_now_add=True)
-    private = db.BooleanProperty()
+class Searches(ndb.Model):
+    user_id = ndb.UserProperty()
+    query_ = ndb.StringProperty()
```

Comment:
Actually, I realise that renaming this has other consequences as well. The users will not be able to see their history after this is deployed, as that's stored in `query` and the code will reference to only `query_`. The solution is to write a wrapper to call `query` method as `query`.